sys-kernel/dracut/dracut-9999.ebuild

```diff
@@ -78,25 +78,19 @@ src_configure() {
 src_install() {
 	default
 
-	local dracutlibdir="usr/lib/dracut"
-
-	local libdirs="/$(get_libdir) /usr/$(get_libdir)"
+	local libdirs=( /$(get_libdir) /usr/$(get_libdir) )
 	if [[ ${SYMLINK_LIB} = yes ]]; then
 		# Preserve lib -> lib64 symlinks in initramfs
-		[[ $libdirs =~ /lib\  ]] || libdirs+=" /lib /usr/lib"
+		[[ ${libdirs[*]} == */lib" "* ]] || libdirs+=( /lib /usr/lib )
```

Comment:
Sorry for not spotting this earlier but wouldn't it be cleaner to just directly use `$(get_libdir)` here?

Reply:
Yes, thanks for spotting this.